README.rst

```diff
@@ -43,14 +43,15 @@ To install `django-import-export-extensions`, run this command in your terminal:
 
     $ pip install django-import-export-extensions
 
-Add `import_export_extensions` to INSTALLED_APPS
+Add `import_export` and `import_export_extensions` to INSTALLED_APPS
```

Comment:
As I mention in other PR's I think we should use  double quotes instead of single ones